src/ServicePulse.Host/vue/src/composables/serviceServiceControl.js

```diff
@@ -94,7 +94,7 @@ export function useServiceControl() {
     //NOTE to eliminate success msg showing everytime the screen is refreshed
     if (newValue != oldValue && !(oldValue === null && newValue === false)) {
       if (newValue) {
-        useShowToast("error", "Error", "Could not connect to ServiceControl at " + serviceControlUrl.value + '. <a class="btn btn-default" href="/configuration#connections">View connection settings</a>');
+        useShowToast("error", "Error", "Could not connect to ServiceControl at " + serviceControlUrl.value + '. <a class="btn btn-default" href="/#/configuration/connections">View connection settings</a>');
```

Comment:
These will still require a full-page refresh. If you're unable to fix that now (the routerlink won't work here) then make sure to create a follow-up issue so that someone else can take a look later.